pkgs/development/python-modules/numcodecs/default.nix

```diff
@@ -2,36 +2,58 @@
   lib,
   stdenv,
   buildPythonPackage,
-  fetchpatch,
   fetchPypi,
+  python,
+  pythonOlder,
+
+  # build-system
   setuptools,
   setuptools-scm,
   cython,
+  py-cpuinfo,
+  pkgconfig,
+
+  # dependencies
   numpy,
+
+  # tests
   msgpack,
-  py-cpuinfo,
   pytestCheckHook,
-  python,
-  pythonOlder,
+  importlib-metadata,
+
+  # buildInputs
+  c-blosc,
+  libzstd,
+  liblz4,
 }:
 
 buildPythonPackage rec {
   pname = "numcodecs";
-  version = "0.13.0";
+  version = "0.13.1";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-uk+scDbqWgeMev4dTf/rloUIDULxnJwWsS2thmcDqi4=";
+    hash = "sha256-o883iB3wiY86nA1Ed9+IEz/oUYW//le6MbzC+iB3Cbw=";
   };
 
+  patches = [
+    # A rebased version of:
+    # https://github.com/zarr-developers/numcodecs/pull/569
+    #
+    # From some reason it fails to apply, even when the PR doesn't merge
+    # conflict:
+    ./system-libs.patch
+  ];
```

Comment:
I think this patch is useful, but I doubt it will be maintained.
In fact, the maintainer is [reluctant to accept it](https://github.com/zarr-developers/numcodecs/pull/569#issuecomment-2323026157) at the moment and there has been no response for 2 months.
So I don't want to accept it now.

Reply:
> I think this patch is useful, but I doubt it will be maintained. In fact, the maintainer is [reluctant to accept it](https://github.com/zarr-developers/numcodecs/pull/569#issuecomment-2323026157) at the moment and there has been no response for 2 months. So I don't want to accept it now.

Hmm I was thinking of maintaining if myself, but I agree it may be too much, and I can maintain that patch on my local overlays.. I sent my word never the less at https://github.com/zarr-developers/numcodecs/pull/569 .